Game/Entities/Bomb.cs

```diff
@@ -0,0 +1,46 @@
+using Godot;
+
+public partial class Bomb : RigidBody2D, IDamageable
+{
+	private float Timeout = 3f;
+	private Timer timer;
+	public override void _Ready()
+	{
+		timer = GetNode<Timer>("ExplosionTimer");
+		timer.WaitTime = Timeout;
+		timer.Start();
+	}
+
+	private int BaseDamage = 50;
+	private void _OnExplosionTimerTimeout()
+	{
+		var area = GetNode<Area2D>("ExplosionRadius");
+		foreach (var body in area.GetOverlappingBodies())
+		{
+			if (body is Enemy enemy)
```

Comment:
If both enemies and players take the same amount of damage, this could just be `if (body is IDamageable damageable)`